Formula/wownero.rb

```diff
@@ -2,18 +2,10 @@ class Wownero < Formula
   desc "Official wallet and node software for the Wownero cryptocurrency"
   homepage "https://wownero.org"
   url "https://git.wownero.com/wownero/wownero.git",
-    tag:      "v0.8.0.2",
-    revision: "7d1693b1601cb4b3d56191953f403c92a1c51f64"
+    tag:      "v0.9.0.0",
+    revision: "b303931b23206236f335bca16b38df8312216668"
   license "BSD-3-Clause"
 
-  bottle do
-    cellar :any
-    rebuild 1
-    sha256 "f68be29309c6d6fe1e9f4d3a96938d9884390e1cd5a94175513f8d6e9a882b8f" => :catalina
-    sha256 "7a828b16ea78869a1ab52d349b2f3a4821b56d28d6a4d726a397bd02cb648868" => :mojave
-    sha256 "544f680ced811219370bd6aa5dc7eb303fa7e90edb771192dbf9528afd09fd25" => :high_sierra
-  end
-
```

Comment:
Please do not remove or change the `bottle` block. Yes, CI will handle updating the values automatically, but it needs the old block to remain.

Reply:
Ok, I placed it back.